schema.json

```diff
@@ -0,0 +1,214 @@
+{
+  "id": "http://api.codeclimate.com/test-coverage-schema#",
+  "$schema": "http://json-schema.org/draft-04/schema#",
+
+  "title": "Test Coverage",
+  "description": "Schema that describes the format of the test coverage payload sent to Code Climate",
+
+  "definitions": {
+    "covered_percent": {
+      "type": "integer",
+      "minimum": 0,
+      "maximum": 100,
+    },
+    "covered_strength": {
+      "type": "integer",
```

Comment:
Similar question: I don't actually know what this number means, but I thought I saw non-whole numbers.